### PR TITLE
[libusb1] Update to 1.0.24. JB#56440

### PR DESCRIPTION
**Pull request by pvuorela**
Removed the static lib sub-package. Assuming it hasn't been needed on
sailfish side.

Changes file removed too. Contained bogus entry and the history was
more of redhat/fedora/moblin.

@Tomin1 @mlehtima @saukko

### CODE REVIEW

File: rpm/libusb1.spec [hunk 1]
```diff
@@ -55,13 +42,9 @@ rm -f $RPM_BUILD_ROOT%{_libdir}/*.la
...
-%doc COPYING
+%license COPYING
 %{_libdir}/*.so.*
 
-%files static
-%defattr(-,root,root,-)
-%{_libdir}/*.a
-
 %files devel
 %doc AUTHORS COPYING README NEWS ChangeLog
```
Comment (Thaodan) [inline rpm/libusb1.spec]: Remove COPYING from here please.

File: rpm/libusb1.spec [hunk 2]
```diff
@@ -17,34 +14,24 @@ BuildRequires: libtool
...
-%configure --enable-static 
+%configure
 # Parallel build fails, thus %{?jobs:-j%jobs} not here.
 make
 
 %install
-rm -rf %{buildroot}
 %make_install
 
 rm -f $RPM_BUILD_ROOT%{_libdir}/*.la
```
Comment (Thaodan) [inline rpm/libusb1.spec]: This is redundant and can be removed.

**Review by Thaodan — CHANGES_REQUESTED** [review]

File: rpm/libusb1.spec (hunk 1 above)
Comment (pvuorela) [reply rpm/libusb1.spec]: Done.

File: rpm/libusb1.spec (hunk 2 above)
Comment (pvuorela) [reply rpm/libusb1.spec]: Yep.

**Review by Thaodan — APPROVED** [review]

File: rpm/libusb1.spec [hunk 3]
```diff
@@ -1,10 +1,7 @@
-%define keepstatic 1
-
 Name:       libusb1
 Summary:    A library which allows userspace access to USB devices
-Version:    1.0.20
+Version:    1.0.24
 Release:    1
-Group:      System/Libraries
 License:    LGPLv2+
 URL:        http://libusb.info/
```
Comment (Thaodan) [inline rpm/libusb1.spec]: this is https://libusb.info/ now

Comment (pvuorela) [reply rpm/libusb1.spec]: Sure, I'll change that too.

**Pull request merged**